lib/src/values/logic_value.dart

```diff
@@ -284,21 +284,24 @@ abstract class LogicValue {
 
   /// Returns a subset [LogicValue].  It is inclusive of [startIndex], exclusive of [endIndex].
   ///
-  /// [startIndex] must be less than [endIndex]. If [startIndex] and [endIndex] are equal, then a
+  /// [startIndex] must come before the [endIndex] on position. If [startIndex] and [endIndex] are equal, then a
   /// zero-width value is returned.
+  /// Negative/Positive index values are allowed. (The negative indexing starts from where the array ends)
   LogicValue getRange(int startIndex, int endIndex) {
```

Comment:
We'll need to update the `operator []` above also, since for `LogicValue` it doesn't reuse the `getRange`